### DISCUSSION TITLE
Browser just keep loading on green screen

### DISCUSSION THREAD

**Discussion by PandicKing** [SafeExamBrowser/seb-win-refactoring#213] [General]
![eQ8xET-Ju84](https://user-images.githubusercontent.com/91611600/135274487-9f8492aa-8989-4e52-a473-1ac09d18f50a.jpg)
I use version 2.4.1 but it's happening with 3x version too

**Comment by dbuechel** [comment]
Version 2.x isn't actively developed anymore, if possible use the latest release version of SEB 3.x. Could you furthermore share what exactly isn't working when trying to start SEB 3.x including a photo and the log files, to be found under `%LocalAppData%\SafeExamBrowser\Logs`.
